.github/workflows/functional_tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Functional tests using Ragger
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  build_appplication:
+    name: Build application using the reusable workflow
+    uses: LedgerHQ/ledger-app-workflows/.github/workflows/build.yml@master
+    with:
+      uploaded_artifact_name: compiled_app_binaries
+
+  functionnal_tests:
+    name: Run ragger tests using the reusable workflow
+    needs: build_appplication
+    uses: LedgerHQ/ledger-app-workflows/.github/workflows/ragger_tests.yml@master
```

Comment:
As far as I know yes, I'll ask for @lpascal-ledger review too I think he's more experienced with this :)

Reply:
That's not custom action, but reusable workflow. But yes, that the way to use it (and it's quite the same for custom actions).
I'm still wondering if we should think of some better versioning that targeting master?
Github default actions (like checkout) uses tag: v1 / v2 / v3. I thing they are using the following flow:
- a new tag is created when there is a compatibility issue
- Else, a new commit is added and the tag is manually moved...

I'm wondering if we should do the same? Maybe based on branchs instead of moving tags?

Reply:
I'll switch to using tags when the workflow is not in draft anymore, I'll keep the conversation open to remind me of this